packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1941,8 +1950,8 @@ export default class Meeting extends StatelessWebexPlugin {
       this.transcription.commandText = data.text ?? '';
     });
 
-    this.webex.internal.voicea.on(VOICEAEVENTS.NEW_CAPTION, async (data) => {
-      await processNewCaptions({data, meeting: this});
```

Comment:
Has the await been removed because of intermediate captions?

Reply:
The `await` is removed because the `processNewCaptions` isn't a method that returns a promise neither is an async function itself. That's why.